reference/api/documents.mdx

```diff
@@ -301,29 +392,79 @@ Delete one document based on its unique id.
 
 You can use this `taskUid` to get more details on [the status of the task](/reference/api/tasks#get-one-task).
 
+## Delete documents by filter
+
+<RouteHighlighter method="POST" route="/indexes/{index_uid}/documents/delete"/>
+
+Delete a selection of documents based on a filter.
```

Comment:
Not a fan of "selection of documents". I'm struggling to justify it, but it feels off here.

```suggestion
Delete a set of documents based on a filter.
```